package.json

```diff
@@ -50,7 +50,8 @@
     "rollup-plugin-postcss": "^4.0.1",
     "rollup-plugin-vue": "^5.1.9",
     "vue": "^2.7.8",
-    "vue-template-compiler": "^2.7.8"
+    "vue-template-compiler": "^2.7.8",
+	"vue-material-design-icons": "^5.1.2",
```

Comment:
on my php this looks fine, not sure why sometimes is shown like this

Reply:
your phpstorm shows tabs as wide as four spaces

the rest of the file is indented with four spaces. your new line uses a tab character.

Reply:
this is a production depenednecy, not a dev dependency. move it.